SourceX/miniwin/misc.cpp

```diff
@@ -120,7 +120,11 @@ WINBOOL DeleteFileA(LPCSTR lpFileName)
 
 bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 {
+    #if defined(WARPUP)
+	if (SDL_Init(SDL_INIT_EVERYTHING & ~(SDL_INIT_HAPTIC|SDL_INIT_CDROM)) <= -1) {

```

Comment:
I'll try and upstream this as it appears to affect multiple SDL 1.2 targets

Reply:
An issue in warpos sdl library : It's unclear if it could affect real Amigas in some situations, but certainly MOS machines running warpos programs, because game stops right there as other test examples showed.

Reply:
It's also affecting Nintendo 3DS

Reply:
Why the need of cdrom then? :)

Reply:
@Cowcat5150 that's why it's going to be upstreamed for all SDL1.2 targets. :P CD is not really needed for devilutionX no.

Reply:
SDL_INIT_CDROM isn't even a thing in SDL2 (the main target)